### ISSUE TITLE
Additional info when pinging ?config

### ISSUE THREAD

**Issue by MansMeg** [statisticssweden/PxWeb#3]
I would like to get the following information from an API:
* All limits (now it seem to be "secret" limits)
* Citation information (how should data be cited)
* Base level and structure. Now I ping "down" through the path to find it. That is usually ~ 4 unnecessary pings.
* languages and versions
* encoding used by the API

Also, it would be great if we didn't count ?config in the rate limit (or have its own limit)

**Comment by MansMeg** [comment]
An additional information that would be good is contact information, say email, if a user finds an error in the API.

**Comment by MansMeg** [comment]
I realized that an additional thing of interest to get from pinging config would be what pxweb version that is used. If this is already available, please ignore this specific request.

**Comment by likp** [comment]
Hi @MansMeg and sorry for the late reply,

But could you please clarify the first and the third point above. We will probably not implement these in the current API but we will try to add them to version 2.0 of the API.

**Comment by MansMeg** [comment]
Sure!

The ”secret” limit comes from that previously it seems like there where different limit of the API. Some where open (accessed through ?config) and some where secret, so there could be a diff between the two. Of course the best would be if there where only one open limit accessable through the API.

The third issue comes from the fact that some users just supply a url somewhere in the API to browse it. Hence it would be great to get the base url for the API to be able to quickly access the base url from any API URL. Does this make sense?

**Comment by likp** [comment]
Yes there have been some confusion of the different type of limits maxValues and maxCells we will see if they can be more clearly expressed.

Ok I see yes that makes sense.
